packages/api/endpoints/granules.js

```diff
@@ -1075,6 +1077,74 @@ async function bulkReingest(req, res) {
   return res.status(202).send({ id: asyncOperationId });
 }
 
```

Comment:
Why is this restricted to the same collection?
If this method is restricted to the same collection, the method and endpoint name should be updated to avoid confusion and misuse.

Reply:
What would you suggest as a solution here?